documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,53 @@
+# TWAP
+
+Unlike the previous tutorial strategies which demonstrated straight-forward tasks like adding configurations and using clock ticks to periodically perform action, this strategy will introduce you to implement a more complicated algorithm. The tools demonstrated in this strategy is : 
+1) Maintaining some states over clock ticks to retain important information that will decide algorithm's next action 
+2) Quantizing order size depending of user's specification of how the user wants order to be executed.
+
+## Overview
+
+TWAP strategy provides the guide for you to add these features : 
+
+* Divide up an order into *n* individual orders
+* Add time delay between each individual orders
+
+Note that unlike Simple Trade strategy which places one order of entire quantity, the number of order is dynamically determined by user's input. Since one order can be placed over a single clock tick, there needs to some sort of information about order 'state' that will indicate when the order should stop.
+
+## Config
+
+To implement a more sophisticated algorithm, you probably want to ask the user for extra information that you need to feed into the algorithm. This can be done by adding more fields to the `config_map` file :
+
+* `time_delay` : Change the question to ask for number of seconds to delay each individual order. (e.g. How many seconds do you want to wait between each individual order?)
+
+* `num_individual_orders` : This is is a new field added to the config map. It should ask for number individual orders that an order should be split up into. (e.g.Into how many individual orders do you want to split this order?)
+
+## Strategy file
+
+Maintain important information about the state when processing orders by adding state variables. Custom state variables can be added to the strategy to by setting variables in the `__init__` function. 
```

Comment:
The introductory paragraph for the code is still diving too fast into the details here. The first sentence should always follow from what was said before, to give people enough context to understand what's going on.

e.g. you should try to remind the reader that the TWAP strategy logic is trying to split a large order into smaller ones over time, and it does that by blah blah blah blah... Remind the reader in plain English what the logic is trying to accomplish first. Then, start talking about why the state variables are added (to track progress across clock ticks). Only then, should you start going into the specifics like the variables in the `__init__` function.

I would also recommend you to re-organize the section to display and explain the flow chart earlier, right after the introductory paragraph of the "Strategy File" section. Explain in plain English why and then what the logic flow is trying to accomplish first. Drill into the specifics like which variable and which functions afterwards.